### DISCUSSION TITLE
FR: Loosen up the templating of product_variants tag

### DISCUSSION THREAD

**Discussion by ncla** [statamic-rad-pack/shopify#126] [Feature Requests]
Right now if you want to change the markup of the `<select>` coming from `product_variants` tag then you must create basically another tag and duplicate this addons tag code. The methods responsible for creating the markup are `private`, and I cannot simply extend the ProductVariants tag class and just override `parseOptions`, where I could add the HTML attribute I want. Ideally, in my opinion, this templating should be moved to antlers or blade templates.

**Comment by jackabox** [comment]
@ncla Sure, it's meant to be quite tight at the moment but I could see the use case. Happy for a PR to tackle this.

**Comment by ryanmitchell** [comment]
I've opened a PR here for that https://github.com/statamic-rad-pack/shopify/pull/173

**Reply by ryanmitchell** [reply]
Included in 2.0.0-beta1
